src/App.svelte

```diff
@@ -1,20 +1,25 @@
 <script>
-  import * as taxData from './lib/data.js';
+//   import * as taxData from './lib/data.js';
+  import taxData from './lib/data.json';
   import * as helper from './lib/helper.js';
 
   let grossIncome = 0;
-  let year = '2022';
-  let marrStatus = 'single';
+  let year = "2022";
+  let marrStatus = "single";
   let totalTaxes = 0;
   let afterTax = 0;
 
   function calcTaxes() {
       // calculate standard deduction and taxable income
-      let standardDeduction = taxData.yearToTax[year]["standardDeduction"][marrStatus];
+      const mapping = "mappings"+year;
```

Comment:
Since "mappings"+year maps to the data we want, and year only maps to "mappings"+year, you should probably just remove "mappings"+year and map the data directly to year instead. Would clean up the data access by a bit.